README.md

```diff
@@ -37,6 +37,13 @@ $ bundle install
 $ jekyll serve
 ~~~
 
+If that fails you may need to run the Jekyll server with, assuming that
+`bundle install` from the last step was ran.
```

Comment:
Yeah, I originally added that to cover cases where bundler was not used to install the depended Gems.